### DISCUSSION TITLE
nordigen import stopped working?

### DISCUSSION THREAD

**Discussion by piotrcichosz** [firefly-iii/firefly-iii#7662] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

my cron tak that was importing successfully for past weeks, so I tried manually importing in data importer panel, but logs show:
`[2023-06-16 06:10:02] local.DEBUG: Request parameters will be set to:
[2023-06-16 06:10:02] local.DEBUG: authenticatedGet(https://ob.nordigen.com/api/v2/accounts/<uuid>/details/?)
[2023-06-16 06:10:18] local.ERROR: GuzzleHttp\Exception\ServerException: Server error: `GET https://ob.nordigen.com/api/v2/accounts/<uuid>/details/` resulted in a `503 Service Unavailable` response:
{"summary":"Couldn't update account details","detail":"Couldn't connect to Institution","status_code":503,"type":"Connec (truncated...)

[2023-06-16 06:10:18] local.ERROR: {"summary":"Couldn't update account details","detail":"Couldn't connect to Institution","status_code":503,"type":"ConnectionError"}
[2023-06-16 06:10:18] local.ERROR: GuzzleHttp\Exception\ServerException: Server error: `GET https://ob.nordigen.com/api/v2/accounts/<uuid>/details/` resulted in a `503 Service Unavailable` response:
{"summary":"Couldn't update account details","detail":"Couldn't connect to Institution","status_code":503,"type":"Connec (truncated...)`

maybe it's a matter of changing domain? nordigen was acquired by gocardless recently. Accounts were migrated, still have access to panel - just under new domain: https://bankaccountdata.gocardless.com/data/
developer quick start guide: https://developer.gocardless.com/bank-account-data/quick-start-guide

### Debug information

`
Debug information generated at 2023-06-16 21:11:15 Europe/Warsaw for Firefly III Data Importer version **1.2.2**.

| Scope | Version |
| --- | --- |
| FIDI | 1.2.2 |
| Build | (unknown), (unknown) |
| PHP | 8.2.4 |
| Host | FreeBSD |

| System info | Value |
| --- | --- |
| Using docker? | false |
| System TZ | Europe/Warsaw |
| Browser TZ | Europe/Warsaw |
| App environment | local |
| App debug mode | true |
| App cache driver | file |
| App logging | debug, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | fpm-fcgi |
| BCscale | 12 |
| Trusted proxies (.env) |  |

| User info | Value |
| --- | --- |
| User agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36 |

`

### Expected behaviour

imported transactions

**Comment by JC5** [comment]
Nordigen has announced no API changes yet. This error simply indicates your bank is down.

**Reply by piotrcichosz** [reply]
I can see transactions when I go to "Developers -> My account data" and select my bank then open transactions. So weird it's available there, but not through the api. 
I will try again later/tomorrow and maybe write to support.

**Reply by piotrcichosz** [reply]
ok, generating new request/connection makes importer working again :)
